### PR TITLE
copy youtube tags to composer keywords

### PR DESCRIPTION
**Pull request by Reettaphant**

### CODE REVIEW

**Comment by Reettaphant** [comment]
@akash1810 This pr has now been updated. There is now a button a separate button for copying composer keywords over and composer keyword tags are matched with the bundle names. I couldn't get react-tooltip working for displaying the tooltip when hovering over the keyword copy button, I'll fix that in a separate pr.

**Comment by Reettaphant** [comment]
Tooltip is now working here as well!

File: app/controllers/YoutubeTagController.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.gu.pandahmac.HMACAuthActions
+import play.api.libs.json.Json
+import play.api.mvc.Controller
+import com.gu.media.youtube.contentBundlingMap
+
+class YoutubeTagController(val authActions: HMACAuthActions) extends Controller {
+
+  import authActions.APIHMACAuthAction
+
+  def getById(id: String) = APIHMACAuthAction { implicit req =>
```
Comment (akash1810) [inline app/controllers/YoutubeTagController.scala]: why `APIHMACAuthAction` over `APIAuthAction`?

File: conf/routes [hunk 2]
```diff
@@ -36,6 +36,7 @@ GET     /api2/pluto/projects/:id        controllers.PlutoProjectController.getPr
 POST    /api2/pluto/projects            controllers.PlutoProjectController.createProject()
 PUT     /api2/pluto/projects/:id        controllers.PlutoProjectController.updateProject(id)
 
+GET     /api2/youtube-tag/:id           controllers.YoutubeTagController.getById(id)
```
Comment (akash1810) [inline conf/routes]: Would `/api2/youtube/content-bundle-tag` be more intention revealing? Also, could this be kept in the `/api2/youtube/` namespace?

File: common/src/main/scala/com/gu/media/youtube/package.scala [hunk 3]
```diff
@@ -11,6 +11,44 @@ import org.joda.time.DateTime
...
+  def contentBundlingMap: Map[String, String] = Map (
+    "uk" -> "gdnpfpnewsuk",
+    "us" -> "gdnpfpnewsus",
+    "au" -> "gdnpfpnewsau",
+    "world" -> "gdnpfpnewsworld",
+    "politics" -> "gdnpfpnewspolitics",
+    "opinion" -> "gdnpfpnewsopinion",
+    "football" -> "gdnpfpsportfootball",
+    "cricket" -> "gdnpfpsportcricket",
+    "rugby union" -> "gdnpfpsportrugbyunion",
```
Comment (akash1810) [inline common/src/main/scala/com/gu/media/youtube/package.scala]: should this be `rugby-union` and `rugby-league`?

File: app/controllers/YoutubeTagController.scala (hunk 1 above)
Comment (Reettaphant) [reply app/controllers/YoutubeTagController.scala]: No reason, just absent mindedness.

File: public/video-ui/src/services/KeywordsApi.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { pandaReqwest } from './pandaReqwest';
+import ContentApi from './capi';
+
+export default {
+  composerTagToYouTube: (tagId) => {
+    const encodedId = encodeURIComponent(tagId)
+    return pandaReqwest({
+      url: '/api2/youtube/content-bundle/' + encodedId,
+      method: 'get'
```
Comment (akash1810) [inline public/video-ui/src/services/KeywordsApi.js]: minor: I think we default as `get`

**Pull request merged**